src/Microsoft.Identity.Client/Internal/Cache/AccessTokenCacheKey.cs

```diff
@@ -27,40 +27,25 @@
 
 using System;
 using System.Collections.Generic;
-using System.Globalization;
 using System.Linq;
 using System.Text;
 
 namespace Microsoft.Identity.Client.Internal.Cache
 {
     /// <summary>
-    /// <see cref="TokenCacheKey" /> can be used with Linq to access items from the TokenCache dictionary.
+    /// <see cref="AccessTokenCacheKey" /> can be used with Linq to access items from the TokenCache dictionary.
```

Comment:
You are right, we do need only ToString(). I would prefer to call it tostring because it is the string representation of the object and more natural to use.